includes/rules/video_present.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Accessibility Checker pluign file.
+ * Accessibility Checker plugin file.
```

Comment:
Good catch. 😉

Reply:
Thanks @SteveJonesDev. Do you want me to merge it?

Reply:
No, I will merge it later. Thanks!